compiler/rustc_target/src/spec/solid_base.rs

```diff
@@ -0,0 +1,21 @@
+use super::LinkerFlavor;
+use crate::spec::{RelroLevel, TargetOptions};
+
+pub fn opts(kernel: &str) -> TargetOptions {
+    TargetOptions {
+        os: format!("solid-{}", kernel),
+        vendor: "kmc".to_string(),
+        exe_suffix: ".out".to_string(),
+        linker_is_gnu: true,
+        linker_flavor: LinkerFlavor::Gcc,
+        has_rpath: false,
+        crt_static_default: true,
+        crt_static_respected: true,
+        has_elf_tls: true,
+        dynamic_linking: true,
+        executables: false,
+        position_independent_executables: true,
```

Comment:
These two are conflicting. `position_independent_executables` only applies to executables and not dynamic libraries, yet you say that executables are not supported.

Reply:
Removed the `position_independent_executables` specification in 66043476105fed3ce828fc493bc30f4ac08d7b75.